### DISCUSSION TITLE
Quiz_23_ Delegatecall 求指正错误

### DISCUSSION THREAD

**Discussion by Hrtypist** [AmazingAng/WTF-Solidity#525] [Q&A] [answered]
1.address
2.C;B;A;B
3.可以指定交易发送的gas，但不可以指定发送的ETH数额
4.变量名、变量类型、声明顺序都必须相同
5.(bool success, bytes memory data) = _addr.delegatecall(abi.encodeWithSignature("mint(uint256)", _num));
6.代理合约; 逻辑合约; 代理合约delegatecall逻辑合约
求大佬指正错误

**Comment by typhoonbb** [comment]
第四题 变量名可以不同，但是原文没有提到这一点，而且感觉solidity这个设定有点古怪

**Reply by XdpCs** [reply]
已经在新版本中，加入了提示

**Comment by XdpCs** [comment, marked as answer]
第四问 变量名可以不同
